### DISCUSSION TITLE
I🔨 Ban Appeal

### DISCUSSION THREAD

**Discussion by JpSolitario016** [MenheraBot/MenheraBot#212] [Ban Appeal]
Qual o seu ID do Discord? 

738904591151923283

Quando foi o seu banimento? 

09/04/2021 Não me lembro :(

Por que tu foi banido? 

Mal Uso Das Sugestões 

Por que tu fez isso? 

Meus amigos mandaram eu sugerir coisas inapropriadas e eu como otario sugeri.

Qual regra que tu quebrou fazendo isso? 

A regra de Mal Sugestões.

Uma mensagem para a Lux 

Lux Eu sei q vc é uma pessoa de bom coração Ent Sei q vai nem ligar pra nd q escrevi aq E nem Vai Remover meu ban mas espero q leia isso kkkkk

Você concorda com os Termos de Uso e Privacidade da Menhera? 

x •  Sim, eu concordo.

**Comment by ySnoopyDogy** [comment]
qual eh o ID do teu discord, tu mandou o discriminator

**Reply by JpSolitario016** [reply]
738904591151923283 Foi Mals Sou Um pouco burro kkkk

**Reply by ySnoopyDogy** [reply]
Desbani pq n lembro quais foram as sugestoes que tu mandou :+1:

**Reply by JpSolitario016** [reply]
Prometo não cometer o mesmo erro!
